doc/build-helpers/build-helpers.chapter.md

```diff
@@ -0,0 +1,106 @@
+# Build helpers {#chap-build-helpers}
```

Comment:
It's odd to have a section with the same title as its parent section. Instead this could be moved to the parent section directly (into `doc/build-helpers.md`)

Reply:
I was just imitating `doc/stdenv/stdenv.chapter.md`. At the time this commit was made, there are still `.xml` files, and neither `doc/stdenv.md` nor `doc/builder.md` exists.

Reply:
Now the introduction goes into `doc/build-helpers.md`, and the sections about fixed-point arguments are promoted as a chapter and placed into `doc/build-helpers/fixed-point-arguments.md`.